ksqldb-functional-tests/src/test/resources/sql-tests/test.sql

```diff
@@ -252,12 +252,4 @@ ASSERT STREAM bar (id INT KEY, col1 VARCHAR) WITH (kafka_topic='BAR', value_form
 CREATE STREAM foo (id INT KEY, col1 INT) WITH (kafka_topic='foo', value_format='JSON');
 CREATE STREAM bar AS SELECT * FROM foo;
 
-ASSERT STREAM bar (id INT KEY, col1 INT) WITH (kafka_topic='BAR', value_format='JSON', wrap_single_value=false);
-
-----------------------------------------------------------------------------------------------------
---@test: assert format doesn't support schema_id property should fail
```

Comment:
Why is this test being removed?

Reply:
When we previously put schema id to format properties. We expect KAFKA not support it and throw exception, now we don't put it there, it will be ignored. It was added in https://github.com/confluentinc/ksql/commit/84e176546b759bcf32f242dd959dfc2a48ce685c. I have next PR to do validation so that we throw exceptions when schema id is provided and format doesn't support schema inference